File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        set<int> mySet;
        for(int n : nums){
            mySet.insert(n);
        }
        int index = 0;
        for(int n :mySet){
            nums[index] = n;
            index++;
        }
        return mySet.size();
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        List<Integer> data = new ArrayList<>();
        data.add(nums[0]);
        //System.out.println(nums[0]);
        int temp = nums[0];
        for(int i=0;i<nums.length;i++){
            if(temp!=nums[i]){
                data.add(nums[i]);
                temp = nums[i];
            }
        }
        for(int i = 0;i<data.size();i++){
            nums[i] = data.get(i);
        }
        return data.size();
    }
}